bin/data-get.js

```diff
@@ -39,7 +39,7 @@ const run = async () => {
     let savedPath
     const parsedIdentifier = await parseDatasetIdentifier(identifier)
     const itIsDataset = isDataset(identifier)
-    if (itIsDataset || parsedIdentifier.type === "datahub" || parsedIdentifier.type === "github") {
+    if (itIsDataset || parsedIdentifier.type === "datahub" || (parsedIdentifier.type === "github" && parsedIdentifier.name.slice((parsedIdentifier.name.lastIndexOf(".") - 1 >>> 0) + 2) === '')) {
```

Comment:
Can you add extensive commentaries on what you're doing here? Otherwise, I don't understand it.

Reply:
I am checking for the file extension, so if it recognises as a file, it will not pass as a dataset.